### PR TITLE
Ea 779-sample-android-app-profile-getProfile-updateProfile_SDK

### PR DESCRIPTION
**Pull request by Arunkumarbwell**

### CODE REVIEW

File: bwell-kotlin-android/local.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
```
Comment (istashak) [inline bwell-kotlin-android/local.properties:1]: This file should be removed from your local git cache and added to .gitignore

File: bwell-kotlin-android/app/build.gradle.kts [hunk 2]
```diff
@@ -69,4 +71,27 @@ dependencies {
     androidTestImplementation("androidx.compose.ui:ui-test-junit4")
     debugImplementation("androidx.compose.ui:ui-tooling")
     debugImplementation("androidx.compose.ui:ui-test-manifest")
+
+
+
+
```
Comment (grantespo) [inline bwell-kotlin-android/app/build.gradle.kts:77]: one space should be ok here

File: bwell-kotlin-android/app/build.gradle.kts [hunk 3]
```diff
@@ -69,4 +71,27 @@ dependencies {
...
+    implementation ("androidx.core:core-ktx:1.12.0")
+    implementation ("androidx.appcompat:appcompat:1.6.1")
+    implementation ("com.google.android.material:material:1.10.0")
+    implementation ("androidx.constraintlayout:constraintlayout:2.1.4")
+    implementation ("androidx.lifecycle:lifecycle-livedata-ktx:2.6.2")
+    implementation ("androidx.lifecycle:lifecycle-viewmodel-ktx:2.6.2")
+    implementation ("androidx.legacy:legacy-support-v4:1.0.0")
+    testImplementation ("junit:junit:4.13.2")
+    androidTestImplementation ("androidx.test.ext:junit:1.1.5")
+    androidTestImplementation ("androidx.test.espresso:espresso-core:3.5.1")
```
Comment (grantespo) [inline bwell-kotlin-android/app/build.gradle.kts:88]: maybe ensure `androidTestImplementation`, `debugImplementation`, and `implementation` are all grouped together

File: bwell-kotlin-android/app/src/main/java/com/bwell/sampleapp/repository/Repository.kt [hunk 4]
```diff
@@ -0,0 +1,332 @@
+package com.bwell.sampleapp.repository
+
```
Comment (istashak) [inline bwell-kotlin-android/app/src/main/java/com/bwell/sampleapp/repository/Repository.kt:2]: You may want to consider an environment switch of some sort in this file. You can create a services directory, with interfaces and implementations of those interfaces -- i.e. Mock and BWell impls. Then based on the environment you can swap implementations.

**Review by istashak — CHANGES_REQUESTED** [review]
In the future, let's try to keep the PR focused around the worked outlined in the Jira ticket. 133 files and all the extra work you've done create an unruly PR for us to review.

**Comment by Arunkumarbwell** [comment]
Closing this PR as it was pointing to the branch which is merger to main branch

**Pull request closed without merging**